02_Lab2_NimishMudhale/bubble_sort.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	n := len(s)
+	sorted := false
+	for !sorted {
+		swapped := false
+		for i := 0; i < n-1; i++ {
+			if s[i] > s[i+1] {
+				s[i+1], s[i] = s[i], s[i+1]
+				swapped = true
+			}
+		}
+		if !swapped {
```

Comment:
You can directly use `swapped` for for loop. When there are no swaps that means the array is sorted.

Reply:
done